corehq/apps/api/resources/v0_5.py

```diff
@@ -1362,6 +1376,24 @@ def _error_code_filter_consumer(self, key, value):
 
         return {"error_code": value}
 
+    def apply_sorting(self, obj_list, options=None):
+        """Custom sort logic to make sure sorting is always applied and object order is consistent
+        between requests. This is required for the cursor pagination."""
```

Comment:
I could do that. I'm not sure how I would do it in a test but I could pass the 'required' order fields to the paginator so that it can validate them during the request.